docs/developer/releasing.md

```diff
@@ -64,10 +10,27 @@ For version v0.x.y:
    > NOTE: `origin` should be the name of the remote pointing to `github.com/kubernetes-sigs/cluster-api`
    - `git push origin v0.x.y`
    - `git push origin test/v0.x.y`
-1. Follow the [Image Promotion process](https://git.k8s.io/k8s.io/k8s.gcr.io#image-promoter) to promote the image from the staging repo to `k8s.gcr.io/cluster-api`
+
+## Promote images from the staging repo to `k8s.gcr.io/cluster-api`
+
+Images are built by the [post push images job](https://testgrid.k8s.io/sig-cluster-lifecycle-image-pushes#post-cluster-api-push-images). This will push the image to a [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api).
+
+1. If you don't have a GitHub token, create one by going to your GitHub settings, in [Personal access tokens](https://github.com/settings/tokens). Make sure you give the token the `repo` scope.
+1. Wait for the above job to complete for the tag commit and for the image to exist in the staging repository, then create a PR to promote the image and tag:
+   - `export GITHUB_TOKEN=<your GH token>`
+   - `make promote-images`
```

Comment:
Do you need to checkout the tag first?

Reply:
You do so that `git describe --abbrev=0 2>/dev/null` gets the right tag for `RELEASE_TAG`, but I'm assuming that if you just created the tag above, you would already have that tag checked out.

Reply:
(alternatively you can always set `RELEASE_TAG` - this might be useful especially if we don't backport this PR)

Reply:
changed the instructions for cutting a tag to export a variable for the release tag so now you definitely don't need to checkout the release tag